tests/unit/merchant.services.test.ts

```
@@ -1,77 +1,96 @@
-import { jest } from '@jest/globals';
 import { mockReset } from 'jest-mock-extended';
 
 // Wait for the mock to be applied
-const { default: prismaMock } = await import('../../src/config/prisma.js') as any;
-const { createMerchant, getMerchant, listMerchants } = await import('../../src/services/merchant.services.js');
+const { default: prismaMock } = (await import('../../src/config/prisma.js')) as any;
+const { createMerchant, getMerchant, listMerchants } = await import(
+  '../../src/services/merchant.services.js'
+);
```

ESLint

[error] Replace `·await·import(⏎··'../../src/services/merchant.services.js'⏎` with `⏎··await·import('../../src/services/merchant.services.js'`

(prettier/prettier)